pkg/controller/storagecluster/quickstarts.go

```diff
@@ -0,0 +1,39 @@
+package storagecluster
+
+import (
+	"context"
+	"fmt"
+	"github.com/ghodss/yaml"
+	"github.com/go-logr/logr"
+	consolev1 "github.com/openshift/api/console/v1"
+	ocsv1 "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func (r *ReconcileStorageCluster) ensureQuickStarts(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
+	for _, qs := range AllQuickStarts {
+		cqs := consolev1.ConsoleQuickStart{}
+		err := yaml.Unmarshal(qs, &cqs)
+		if err != nil {
+			reqLogger.Info(fmt.Sprintf("failed to unmarshal consolequickstarts, name: %s, namespace: %s", cqs.Name, cqs.Namespace))
+			return err
+		}
+		found := consolev1.ConsoleQuickStart{}
+		err = r.client.Get(context.TODO(), types.NamespacedName{Name: cqs.Name, Namespace: cqs.Namespace}, &found)
+		if err != nil {
+			if errors.IsNotFound(err) {
+				reqLogger.Info(fmt.Sprintf("Creating Quickstarts, name: %s, namespace: %s", cqs.Name, cqs.Namespace))
+				return r.client.Create(context.TODO(), &cqs)
```

Comment:
Shouldn't this attempt to process the rest of the AllQuickStarts? This should return only in case of failure.

Reply:
added error and return on failure